### PR TITLE
Improves machinepool docs

### PR DESCRIPTION
**Pull request by KhizerJaan**

### CODE REVIEW

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-dee7ccc8`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-bf7dea1b`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-b81d132f`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-7bdaa02e`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-6eb8c22b`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-43d294e7`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-10871e33`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-b963fb32`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-00786058`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-c1832d93`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-7446e65c`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-dfa263be`

File: docs/content/sre/machinepools/machinepools.md [hunk 1]
```diff
@@ -4,60 +4,73 @@ A machine pool creates machine sets that are all clones of the same configuratio
...
-
-## 2x Worker
-
-In a Kubernetes cluster, the worker nodes are where the actual workloads requested by Kubernetes users run and are managed. The worker nodes advertise their capacity and the scheduler, which is part of the master services, determines on which nodes to start containers and pods. We divide worker machines into infra,logging,monitoring and pipeline machines to have dedicated machines for every type of resources. Two worker machines are used.
-
-|   | vCPU | Memory |  Cost/mo<sup id="fnref1"><a href="#fn1" rel="footnote">*</a></sup>|
-|---|---|---|---|
-| Amazon EC2  | 8  | 32 GiB  | 300 USD  |
-|  Binero | 8 | 32 GB  | 2079.88 SEK  |
+The control plane, which is composed of control plane machines (also known as the master machines), manages a SAAP cluster. Exactly three control plane nodes must be used for all production deployments. The control plane machines manage workloads on the compute machines, which are also known as worker machines.
```
Comment (karl-johan-grahn) [inline docs/content/sre/machinepools/machinepools.md]: Exactly three or minimum three, as long as they are odd numbered?

File: docs/content/sre/machinepools/machinepools.md [hunk 2]
```diff
@@ -4,60 +4,73 @@ A machine pool creates machine sets that are all clones of the same configuratio
...
+
```
Comment (karl-johan-grahn) [inline docs/content/sre/machinepools/machinepools.md:26]: We need to specify if this is min requirement per machin, clarify!

File: docs/content/sre/machinepools/machinepools.md [hunk 3]
```diff
@@ -4,60 +4,73 @@ A machine pool creates machine sets that are all clones of the same configuratio
...
+| vCPU | Memory |
+|---|---|
+| 4  | 32 GiB  |
+
+## 1x Infra
+
+The following infrastructure workloads do not incur SAAP worker subscriptions, at least one infrastructure node is required for deployments.
```
Comment (karl-johan-grahn) [inline docs/content/sre/machinepools/machinepools.md]: You talk about two different things, costs and min required nodes, use two separate sentences.

do not incure SAAP worker subscription _costs_

File: docs/content/sre/machinepools/machinepools.md [hunk 4]
```diff
@@ -4,60 +4,73 @@ A machine pool creates machine sets that are all clones of the same configuratio
...
+
+|  | vCPU | Memory |
+|---|---|---|
+| Ingress monitor controller  | 150m  | 600 MiB  |
+| OpenShift-GitOps  | 530 m  | 500 MiB  |
+| Stakater-nexus  | 200 m  | 1.6 GiB  |
+| Vault  | 255 m  | 360 MiB  |
+|  Stakater-Tronador  | 100 m  | 200 MiB  |
+|  OpenShift-Velero  | 500 m  | 150 MiB  |
+|  Stakater-tenant-operator  | 600 m  | 1.2 GiB  |
```
Comment (karl-johan-grahn) [inline docs/content/sre/machinepools/machinepools.md]: Multi Tenant Operator

File: docs/content/sre/machinepools/machinepools.md [hunk 5]
```diff
@@ -4,60 +4,73 @@ A machine pool creates machine sets that are all clones of the same configuratio
...
+|  Stakater-Forecastle  | 50 m  | 200 MiB  |
+|  Stakater-SonarQube  | 350 m  | 1.5 GiB  |
```
Comment (karl-johan-grahn) [inline docs/content/sre/machinepools/machinepools.md]: Why Stakater-SonarQube but just Vault?

File: docs/content/sre/machinepools/machinepools.md [hunk 6]
```diff
@@ -4,60 +4,73 @@ A machine pool creates machine sets that are all clones of the same configuratio
...
+
+| vCPU | Memory |
+|---|---|
+| 4  | 16 GiB  |
+## 1x Logging
 
-## 3x Infra
+For Cluster aggregated logging , we offer one dedicated logging machine. Minimum requirements for logging infrastructure is as follows:
```
Comment (karl-johan-grahn) [inline docs/content/sre/machinepools/machinepools.md]: we offer or we require?

File: docs/content/sre/machinepools/machinepools.md [hunk 7]
```diff
@@ -4,60 +4,73 @@ A machine pool creates machine sets that are all clones of the same configuratio
...
-We use three infrastructure machines for all the resources except for logging , monitoring or pipelines workloads.
+|  | vCPU | Memory |
+|---|---|---|
+| OpenShift logging | 4  | 12 GiB  |
 
-|   | vCPU | Memory |  Cost/mo<sup id="fnref1"><a href="#fn1" rel="footnote">*</a></sup> |
-|---|---|---|---|
-| Amazon EC2  | 8  | 32 GiB  | 300 USD  |
-|  Binero | 8 | 32 GB  | 2079.88 SEK  |
+| vCPU | Memory |
```
Comment (karl-johan-grahn) [inline docs/content/sre/machinepools/machinepools.md]: why is this here?

File: docs/content/sre/machinepools/machinepools.md [hunk 8]
```diff
@@ -4,60 +4,73 @@ A machine pool creates machine sets that are all clones of the same configuratio
...
+|---|---|
+| 4  | 16 GiB  |
 
-## 1x Logging
+## 1x Monitoring
 
-One logging machine is used for cluster logging workloads.
+SAAP exposes metrics that can be collected and stored in back-ends by the cluster-monitoring-operator. As an SAAP administrator, you can view system resources, containers and components metrics in one dashboard interface, Grafana. Exactly one monitoring nodes must be used for all production deployments. The recommended size of a monitoring node is the minimum of 6 vCPUs and 14 GB ram per 7200 pods.
```
Comment (karl-johan-grahn) [inline docs/content/sre/machinepools/machinepools.md]: As _a_ SAAP admin

Exactly one monitoring _node_

Why do we specify recommended size here but not under other sections?

File: docs/content/sre/machinepools/machinepools.md [hunk 9]
```diff
@@ -4,60 +4,73 @@ A machine pool creates machine sets that are all clones of the same configuratio
...
-|---|---|---|---|
-| Amazon EC2  | 4  | 16 GiB  | 150 USD  |
-|  Binero | 4  | 16 GB | 880.04 SEK  |
+|  | vCPU | Memory |
+|---|---|---|
+| OpenShift monitoring | 600 m  | 3 GiB  |
+| Stakater-workload-monitoring | 1 GiB  | 4 GiB  |
 
-## 2x Monitoring
+| vCPU | Memory |
```
Comment (karl-johan-grahn) [inline docs/content/sre/machinepools/machinepools.md]: redundant?

File: docs/content/sre/machinepools/machinepools.md [hunk 10]
```diff
@@ -4,60 +4,73 @@ A machine pool creates machine sets that are all clones of the same configuratio
...
 
-Two monitoring machines are used for cluster monitoring workloads
 
-|   | vCPU | Memory |  Cost/mo<sup id="fnref1"><a href="#fn1" rel="footnote">*</a></sup> |
-|---|---|---|---|
-| Amazon EC2  |8  | 32 GiB  | 300 USD  |
-|  Binero | 6  | 32 GB  | 1640.09 SEK  |
+## 1x Pipeline
 
+Red Hat OpenShift Pipelines is a cloud-native, continuous integration and continuous delivery (CI/CD) solution based on Kubernetes resources. You can use the SAAP Developer console to create Tekton resources, view logs of pipeline runs, and manage pipelines in your SAAP namespaces. We offer one dedicated pipeline machine. Minimum requirements for pipeline infrastructure is as follows:
```
Comment (karl-johan-grahn) [inline docs/content/sre/machinepools/machinepools.md]: Once again, we offer or we require?

Also link to other doc sources if there exists

File: docs/content/sre/machinepools/machinepools.md [hunk 11]
```diff
@@ -4,60 +4,73 @@ A machine pool creates machine sets that are all clones of the same configuratio
...
+|---|---|
+| 4  | 16 GiB  |
 
-|   | vCPU | Memory |  Cost/mo<sup id="fnref1"><a href="#fn1" rel="footnote">*</a></sup> |
-|---|---|---|---|
-| Amazon EC2  | 4  | 16 GiB  | 150 USD  |
-|  Binero | 6  | 24 GB  | 1320.06 SEK  |
+## 3x Worker
 
+In a SAAP cluster, the worker nodes are where the actual workloads requested by users run and are managed. The worker nodes advertise their capacity and the scheduler, which is part of the master services, determines on which nodes to start containers and pods. Three worker machines are used.
```
Comment (karl-johan-grahn) [inline docs/content/sre/machinepools/machinepools.md]: Three worker nodes are used? Three are required? Three are the minimum? Three are the suggested?

**Review by karl-johan-grahn — DISMISSED** [review]

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-ff1150ef`

File: docs/content/sre/machinepools/machinepools.md (hunk 6 above)
Comment (KhizerJaan) [reply docs/content/sre/machinepools/machinepools.md]: we offer , its optional

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-47403f15`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-b7059261`

File: docs/content/sre/machinepools/machinepools.md [hunk 12]
```diff
@@ -4,60 +4,95 @@ A machine pool creates machine sets that are all clones of the same configuratio
...
+Specifications for logging nodes are as follows :
 
-|   | vCPU | Memory |  Cost/mo<sup id="fnref1"><a href="#fn1" rel="footnote">*</a></sup> |
-|---|---|---|---|
-| Amazon EC2  | 4  | 16 GiB  | 150 USD  |
-|  Binero | 6  | 24 GB  | 1320.06 SEK  |
+| vCPU | Memory |
+|---|---|
+| 4  | 16 GiB  |
+## 1x Pipeline(optional)
```
Comment (karl-johan-grahn) [inline docs/content/sre/machinepools/machinepools.md]: space btw pipeline and optional

File: docs/content/sre/machinepools/machinepools.md [hunk 13]
```diff
@@ -4,60 +4,95 @@ A machine pool creates machine sets that are all clones of the same configuratio
...
-Two monitoring machines are used for cluster monitoring workloads
+| vCPU | Memory |
+|---|---|
+| 4 | 16 GiB  |
 
-|   | vCPU | Memory |  Cost/mo<sup id="fnref1"><a href="#fn1" rel="footnote">*</a></sup> |
-|---|---|---|---|
-| Amazon EC2  |8  | 32 GiB  | 300 USD  |
-|  Binero | 6  | 32 GB  | 1640.09 SEK  |
+## 1x Logging (optional)
```
Comment (karl-johan-grahn) [inline docs/content/sre/machinepools/machinepools.md]: clarify why optional groups are optional

File: docs/content/sre/machinepools/machinepools.md [hunk 14]
```diff
@@ -4,60 +4,95 @@ A machine pool creates machine sets that are all clones of the same configuratio
...
+Pipeline machine holds pods running for pipelines. Minimum requirements for pipeline infrastructure is as follows: 
 
-<h4 id="fn1">
-    <a href="#fn1" rel="footnote">*  </a>This estimate is an approximation and may differ.
-</h4>
+|  | vCPU | Memory |
+|---|---|---|
+| OpenShift pipelines | 100 m  | 200 MiB  |
+
+Specifications for pipeline nodes are as follows :
```
Comment (karl-johan-grahn) [inline docs/content/sre/machinepools/machinepools.md]: Specifications or minimum requirements, clarify

File: docs/content/sre/machinepools/machinepools.md [hunk 15]
```diff
@@ -4,60 +4,95 @@ A machine pool creates machine sets that are all clones of the same configuratio
...
+
+| vCPU | Memory |
+|---|---|
+| 4  | 16 GiB  |
+
+## 3x Worker
+
+In a SAAP cluster, the worker nodes are where the actual application workloads run and are managed. Minimum three worker machines are required for SAAP deployments.
+
```
Comment (karl-johan-grahn) [inline docs/content/sre/machinepools/machinepools.md:114]: once again, clarify what this is

**Review by karl-johan-grahn — DISMISSED** [review]

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-7e7e755b`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-d06ed36b`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-aaa738f4`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-539e8bab`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-3129cc7a`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-ce93cde7`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-792bebbf`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-6785b2f9`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-84e22029`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-021d25d9`

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-80736312`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-feab00b6`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-e394a141`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-74dd390b`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-3c35e68b`

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-32913bfa`

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-273089ef`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-d1938e7e`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-8bda556e`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-03cab9bb`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-abc24f9f`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-6ef1b7f4`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-838d3261`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-3a904aa3`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-1deda6fe`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-69f1e946`

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-f0c69328`

**Comment by stakater-user** [comment]
@usamaahmadkhan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-70b46207`

**Comment by stakater-user** [comment]
@usamaahmadkhan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-d56924a9`

**Comment by stakater-user** [comment]
@usamaahmadkhan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-85-925a702a`

**Review by karl-johan-grahn — APPROVED** [review]

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.58`
